_about/notes/0.6.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Istio 0.6
+
+order: 95
+
+layout: about
+type: markdown
+---
+{% include home.html %}
+
+In addition to the usual pile of bug fixes and perf improvements, this release includes the new or 
+updated features detailed below.
+
+## Networking
+
+- **Custom Envoy Config**. Pilot now supports ferrying custom Envoy config to the
+proxy. TBD: doc link?
+
+## Mixer adapters
+
+- **SolarWinds**. Mixer can now interface to AppOptics and Papertrail.
+[Learn more]({{home}}/docs/reference/config/adapters/solarwinds.html)
+
+- **Redisquota**. Mixer now supports a Redis-based adapter for rate limit tracking.
+[Learn more]({{home}}/docs/reference/config/adapters/redisquota.html)
+
+- **Datadog**. Mixer now provides an adapter to deliver metric data to a Datadog agent.
+[Learn more]({{home}}/docs/reference/config/adapters/datadog.html)
+
+## Other
+
+- **Separate Check & Report Clusters**. When configuring Envoy, it's now possible to use different clusters
+for Mixer instances that are used for Mixer's Check functionality from those used for Mixer's Report
+functionality. This may be useful in large deployments for better scaling of Mixer instances.
+TBD: doc link?
+
+- **Monitoring Dashboards**. There are now preliminary Mixer & Pilot monitoring dashboard in Grafana.
+
+- **Servicegraph Visualization**. Servicegraph has a new visualization. [Learn more]({{home}}/docs/tasks/telemetry/servicegraph.html).
+
+- **Liveness and Readiness Probes**. Istio components now provide canonical liveness and readiness
+probe support to help ensure mesh infrastructure health. [Learn more]({{home}}/docs/tasks/security/health-check.html)
+
+- **Egress Policy and Telemetry**. TBD: What to can we say? TBD: doc link?
```

Comment:
Istio can monitor traffic to external services defined by EgressRule or External Service. Istio can also apply mixer policies on this traffic.

Detailed doc not ready yet.